lighthouse-core/scripts/gcp-collection/run.sh

```diff
@@ -12,7 +12,7 @@ DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 LH_ROOT="$DIRNAME/../../.."
 cd $DIRNAME
 
-GCLOUD_USER=$(gcloud config get-value account | awk -F '@' '{print $1}')
+GCLOUD_USER=$(gcloud config get-value account | awk -F '@' '{gsub("[^a-z]","",$1); print $1}')
```

Comment:
this awk went from readable to unintelligible in such short order :(

Reply:
ha well you're welcome to propose an alternative that removes invalid characters :) (scripts wouldn't run because my gcloud user is `patrick.hulce`)